src/main/java/io/github/fusionflux/portalcubed/mixin/props/ThrowPotionMixin.java

```diff
@@ -0,0 +1,49 @@
+package io.github.fusionflux.portalcubed.mixin.props;
+
+import com.llamalad7.mixinextras.sugar.Local;
+
+import io.github.fusionflux.portalcubed.content.PortalCubedEntities;
+import io.github.fusionflux.portalcubed.content.prop.entity.Prop;
+import io.github.fusionflux.portalcubed.data.tags.PortalCubedEntityTags;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.entity.projectile.ThrowableItemProjectile;
+import net.minecraft.world.entity.projectile.ThrownPotion;
+
+import net.minecraft.world.level.Level;
+import net.minecraft.world.phys.AABB;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.function.Predicate;
+
+@Mixin(ThrownPotion.class)
+public abstract class ThrowPotionMixin extends ThrowableItemProjectile  {
+	@Unique
+	private static final Predicate<Prop> IS_PROP = e -> e.getType().is(PortalCubedEntityTags.CAN_BE_WASHED) && PortalCubedEntities.PROPS.containsValue(e.getType());
```

Comment:
Just inline this instead of a field, and checking if the entity type is in the map is useless because you're only getting entities extending `Prop`